src/utils/url.ts

```diff
@@ -125,7 +125,7 @@ export function getURLString (url: string): string {
 }
 
 export function getProxyUrl (url: string, opts: ProxyUrlOptions) {
-    let params: any = [opts.sessionId];
+    const params: Array<string> = [opts.sessionId];
```

Comment:
Stringify url inside the `getProxyUrl` function and simplify this code line - https://github.com/DevExpress/testcafe-hammerhead/blob/630c488aa2f31d721baf86151c7758f25264baec/src/client/sandbox/node/window.ts#L630